docs/workflows/genomic_characterization/freyja.md

```diff
@@ -44,15 +44,29 @@ The main workflow is **Freyja_FASTQ_PHB** ([Figure 1](freyja.md#figure1)). Depen
         - Illumina Paired-End
         - Oxford Nanopore
 
-**Freyja_Update_PHB** will copy the SARS-CoV-2 reference files (`curated_lineages.json` and `usher_barcodes.feather`) from [the source repository](https://github.com/andersen-lab/Freyja/tree/main/freyja/data) to a user-specific Google Cloud Storage (GCP) location (often a [Terra.bio](http://Terra.bio) workspace-associated bucket). These files can then be used as input for the Freyja_FASTQ_PHB workflow.
+[**Freyja_Update_PHB**](freyja.md#freyja_update) will copy the **SARS-CoV-2** reference files that can then be used as input for the [Freyja_FASTQ_PHB](freyja.md#freyja_fastq) workflow.
 
-Two options are available to visualize the Freyja results: **Freyja_Plot_PHB** and **Freyja_Dashboard_PHB.** Freyja_Plot_PHB aggregates multiple samples using output from Freyja_FASTQ_PHB to generate a plot that shows fractional abundance estimates for all samples. including the option to plot sample collection date information. Alternatively, **Freyja_Dashboard_PHB** aggregates multiple samples using output from Freyja_FASTQ to generate an interactive visualization. This workflow requires an additional input field called viral load, which is the number of viral copies per liter.
+Two options are available to visualize the Freyja results: [**Freyja_Plot_PHB**](freyja.md#freyja_plot) and [**Freyja_Dashboard_PHB**](freyja.md#freyja_dashboard). [Freyja_Plot_PHB](freyja.md#freyja_plot) aggregates multiple samples using output from [Freyja_FASTQ_PHB](freyja.md#freyja_fastq) to generate a plot that shows fractional abundance estimates for all samples. including the option to plot sample collection date information. Alternatively, [**Freyja_Dashboard_PHB**](freyja.md#freyja_dashboard) aggregates multiple samples using output from [Freyja_FASTQ_PHB](freyja.md#freyja_fastq) to generate an interactive visualization. This workflow requires an additional input field called viral load, which is the number of viral copies per liter.
 
-## Freyja_Update_PHB {#freyja_update}
+### Freyja, Sequencing Platforms and Data Quality
 
-This workflow will copy the Freyja reference files (`usher_barcodes.feather` and `curated_lineages.json`) to a GCP URI of your choice for usage in Freyja_FASTQ_PHB.
+The choice of sequencing platform and the quality of the data directly influence Freyja's performance. High-accuracy platforms like Illumina provide reliable SNV detection, enhancing the precision of lineage abundance estimates. In contrast, platforms with higher error rates, such as Nanopore, whislt it has improved greatly in the recent years, may introduce uncertainties in variant calling, affecting the deconvolution process. Sequencing depth requirements will increase as the quality of the sequencing data decreases. A rational target depth is 100X coverage for sequencing data with Q-scores in the range of 25-30.
```

Comment:
typo in whilst

"In contrast, platforms with higher error rates, such as Nanopore, **whislt** it has improved greatly in the recent years, may introduce..."

Reply:
Opssss!